production.Dockerfile

```diff
@@ -135,23 +135,20 @@ RUN apt-get update && \
     ( curl -s -L "https://mmdbcdn.posthog.net/" --http1.1 | brotli --decompress --output=./share/GeoLite2-City.mmdb ) && \
     chmod -R 755 ./share/GeoLite2-City.mmdb
 
-
 #
 # ---------------------------------------------------------
 #
-# Build a version of the unit docker image for python3.10
-# We can remove this step once we are on python3.11
-FROM unit:python3.11 as unit
-FROM python:3.10-bullseye as unit-131-python-310
```

Comment:
Everything between here and line 236 can be removed now, and we can just use the unit:python3.11 image directly